src/jit/emitxarch.h

```diff
@@ -40,10 +40,11 @@ struct CnsVal
 };
 
 UNATIVE_OFFSET emitInsSize(code_t code);
-UNATIVE_OFFSET emitInsSizeRM(instruction ins);
```

Comment:
As far as I can tell, this was previously unused? I was trying to figure out how the new `emitInsSizeRR` overloads replaced it.

Reply:
Right, `git grep` showed zero usages.